tests/test_pixel.py

```diff
@@ -1037,28 +1046,26 @@ def test_single_image():
     else:
         config['verbose'] = 0
 
-    try:
-        import jax
-        jax.config.update("jax_enable_x64", True)
-    except ImportError:
-        print("JAX not installed.")
+    configSolver = copy.deepcopy(config)
+    
+    test_star_solver = []
 
-    configJax = copy.deepcopy(config)
-    configJax['psf']['interp']['use_jax'] = True
-    test_star_jax = []
+    solvers = ["scipy", "qr", "jax", "cpp", "cpp32"]
+    rtols = [1e-7, 1e-7, 1e-7, 2e-1]
```

Comment:
Um.  2e-1 is very high tolerance.  Is the cpp solver really that far wrong?  If so, that's a problem I think.  And should include cpp32 to at least investigate how wrong that is as well.

Reply:
Here I compared everything to `"scipy"` solver, so the `rtol=1e-7` is for `"qr"`, `"jax"`, and `"cpp"` compared to `"scipy"` solution. The `rtol=2e-1` is for `"cpp32"` solver that as expected is giving worst results. 

In addition of those test, I had a notebook on the side to make sure that `"scipy" == "jax" == "cpp"`.  `"cpp32"` is not

Reply:
I see.  I missed how the +1 was applied.  If cpp32 is only correct at 20% tolerance, I don't think it's worth keeping.  That's really quite bad.

Reply:
Sounds good to me, I removed it.